galley/pkg/config/analysis/analyzers/gateway/certificate.go

```diff
@@ -40,7 +41,7 @@ func (*CertificateAnalyzer) Metadata() analysis.Metadata {
 
 // Analyze implements analysis.Analyzer
 func (gateway *CertificateAnalyzer) Analyze(context analysis.Context) {
-	scopeGatewayToNamespace := getScopeGatewayToNamespace(context)
+	scopeGatewayToNamespace := getScopeGatewayToNamespace()
```

Comment:
No need for new func just to return a value.
```suggestion
	scopeGatewayToNamespace := features.ScopeGatewayToNamespace
```

or directly write
```
gateway.analyzeDuplicateCertificate(resource, context, features.ScopeGatewayToNamespace)
```

Reply:
@shamsher31 Done. Thanks for the suggestion!